language/move-command-line-common/src/values.rs

```diff
@@ -215,6 +217,29 @@ impl Token for ValueToken {
                 }
                 (ValueToken::ByteString, len)
             }
+            's' if matches!(chars.peek(), Some('"')) => {
```

Comment:
I'm fine with just no prefix here. Or does that break things?

Reply:
Done (does not seem to breaking anything!)